action.yml

```diff
@@ -167,6 +167,22 @@ runs:
         sudo ln -svf caproto-repeater "/bin/caRepeater"
         echo "::endgroup::"
 
+    - name: Apply auto settings
+      shell: bash -leo pipefail {0}
+      run: |
+        echo "::group::Applying auto settings"
+        if [ -f "${CLONED_REPO}/.ci/auto_settings.sav" ]; then
+          while IFS= read -r line; do
+            if [ -n "$line" ]; then
+                echo "Applying $line"
+                caput $line
```

Comment:
Do we have `caput` available? Probably it needs to be installed. Is there a test for that somewhere?

Reply:
Ah probably not, I think we will require that the conda environment be activated prior to this step.

Reply:
@mrakitin I still get the error even after activating the conda environment. I tried using the same environment locally and indeed `caput` is not available even though `epics-base` is installed via conda-forge. I think its a `PATH` issue from downloading the environment.

When I manually do
```bash
conda install -c conda-forge epics-base -y
```
Then `caput` works. Is this a problem with the environment we get from zenodo?